scripts/ci/deploy/config/databend-query-node-1.toml

```diff
@@ -110,3 +109,27 @@ data_path = "./.databend/stateless_test_data"
 # endpoint_url = "<your-endpoint>"
 # access_key_id = "<your-key-id>"
 # access_key_secret = "<your-account-key>"
+
+
+[cache]
+
+# in-memory cache of ....
+table_meta_enabled = true
+table_meta_snapshot_item = 1024
+table_meta_segment_item = 1024
+
+table_index_bloom_enabled = true
+table_index_bloom_meta = 1024
```

Comment:
What's the unit?
`table_index_bloom_meta` -> `table_index_bloom_meta_max_[unit]`?

It would be better to add `min/max` and the `unit` to the config item, will be more readable.

Reply:
the new config entries are here :
https://github.com/datafuselabs/databend/pull/9772/files#diff-05f47a02db443a04d8f1592207d6cee96c45b76563eb99d6a97ca27403fac099R1900-R1914